vtkLookingGlassInterface.cxx

```diff
@@ -627,6 +628,132 @@ void vtkLookingGlassInterface::GetTilePosition(int tile, int pos[2])
   pos[1] = (tile / this->QuiltTiles[0]) * this->RenderSize[1];
 }
 
+void vtkLookingGlassInterface::RenderQuilt(
+  vtkOpenGLRenderWindow* rw, std::function<void(void)>* renderFunc)
+{
+  vtkCollectionSimpleIterator rsit;
+
+  // loop over the tiles, render,and blit
+  vtkOpenGLFramebufferObject* renderFramebuffer;
+  vtkOpenGLFramebufferObject* quiltFramebuffer;
+  this->GetFramebuffers(rw, renderFramebuffer, quiltFramebuffer);
+
+  auto ostate = rw->GetState();
+
+  ostate->PushFramebufferBindings();
+  renderFramebuffer->Bind(GL_READ_FRAMEBUFFER);
+
+  int renderSize[2];
+  this->GetRenderSize(renderSize);
+
+  int tcount = this->GetNumberOfTiles();
+
+  // save the original camera settings
+  auto* renCol = rw->GetRenderers();
+  vtkRenderer* aren;
+  std::vector<vtkCamera*> Cameras;
+  for (renCol->InitTraversal(rsit); aren = renCol->GetNextRenderer(rsit);)
+  {
+    // Ugly piece of code - we need to know if the camera already
+    // exists or not. If it does not yet exist, we must reset the
+    // camera here - otherwise it will never be done (missing its
+    // oppportunity to be reset in the Render method of the
+    // vtkRenderer because it will already exist by that point...)
+    if (!aren->IsActiveCameraCreated())
+    {
+      aren->ResetCamera();
+    }
+    auto oldCam = aren->GetActiveCamera();
+    oldCam->SetLeftEye(1);
+    oldCam->Register(rw);
+    Cameras.push_back(oldCam);
+    vtkNew<vtkCamera> newCam;
+    aren->SetActiveCamera(newCam);
+  }
+
+  // Make sure we can use offscreen buffers
+  auto prevUseOffScreenBuffers = rw->GetUseOffScreenBuffers();
+  rw->UseOffScreenBuffersOn();
```

Comment:
This was only done in ParaView, but I assumed it is safe to leave for all three cases.

Reply:
@martinken I can move this outside of the function and use it just in ParaView if you think that's better.

Reply:
I removed this from `RenderQuilt()` and put it back in the ParaView version only, so we can ensure the VTK implementations have the same behavior as before.